### DISCUSSION TITLE
AXIS Model Q6125-LE Remote control

### DISCUSSION THREAD

**Discussion by gpaggi2** [AxisCommunications/discussions#228] [VAPIX] [answered]
Dear all,
I am a system integrator. I am new to AXIS camera.

I've got this camera:
Brand= AXIS
Model= Q6125-LE

and I would like to know

1) Which URL should I use to be able to show the video streaming (codec MJPEG, protocol HTTP). It should be something simlar to this.

http://user:password@192.168.140.31/cgi-bin/mjpeg)

2) Which protocol should I use to control the pan, tilt, zoom of camera from a remote PC (on the same network) which is running a software developed on Python/C++. Is there any specific protocol and/or documentation and a library to be used? I am already familiar with CGI protocols

Best regards
    GP

**Comment by onie5** [comment, marked as answer]
Hello @gpaggi2,

1. You can use a VAPIX commands to pull MJPEG stream, check this library link:

https://www.axis.com/vapix-library/subjects/t10175981/section/t10051110/display?section=t10051110-t10004103

2. You use curl with the commands from this VAPXI library:

https://www.axis.com/vapix-library/subjects/t10175981/section/t10036011/display?section=t10036011-t10004639

Cheers.
